### PR TITLE
Rename `Params` to `Config` & fix test code to new ff

### PR DESCRIPTION
**Pull request by sunhuachuang**
## Description

Upgrade to the latest algebra: https://github.com/arkworks-rs/algebra/pull/393

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (master)
- [x] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [ ] Wrote unit tests
- [x] Updated relevant documentation in the code
- [ ] Added a relevant changelog entry to the `Pending` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: src/bits/uint8.rs [hunk 1]
```diff
@@ -335,7 +335,7 @@ impl<ConstraintF: Field> AllocVar<u8, ConstraintF> for UInt8<ConstraintF> {
     }
 }
 
-/// Parses the `Vec<UInt8<ConstraintF>>` in fixed-sized `ConstraintF::Params::CAPACITY` chunks and
+/// Parses the `Vec<UInt8<ConstraintF>>` in fixed-sized `ConstraintF::MODULUS_BIT_SIZE` chunks and
```
Comment (Pratyush) [inline src/bits/uint8.rs:338]:
```suggestion
/// Parses the `Vec<UInt8<ConstraintF>>` in fixed-sized `ConstraintF::MODULUS_BIT_SIZE - 1` chunks and
```

**Comment by Pratyush** [comment]
This looks good to me; @weikengchen should we merge this into your branch?

**Comment by weikengchen** [comment]
Let me take a look. I told Sun to disregard my PR and do the necessary changes all in this one.

File: src/bits/uint8.rs (hunk 1 above)
Comment (sunhuachuang) [reply src/bits/uint8.rs]: Fixed.

**Review by Pratyush — APPROVED** [review]

**Comment by Pratyush** [comment]
Sounds good. Should I merge these into your branch, @weikengchen , or into main?

**Comment by Pratyush** [comment]
Put otherwise, @sunhuachuang, if we point this branch to the appropriate branch on `curves`, do things compile and pass tests?

**Comment by weikengchen** [comment]
I think let us merge it to `fix-ff` and see if it works. 

Sun is in China and will be up in a few hours.

**Pull request merged**